ui/tests/acceptance/secrets/backend/aws/aws-configuration-test.js

```diff
@@ -89,9 +91,8 @@ module('Acceptance | aws | configuration', function (hooks) {
       await enablePage.enable('aws', path);
 
       this.server.post(configUrl('aws-lease', path), () => {
-        assert.false(
-          true,
-          'post request was made to config/lease when no data was changed. test should fail.'
+        throw new Error(
```

Comment:
👏 This is much easier for me to follow what's happening here! Thanks for updating